src/hardware/input/mouse_config.cpp

```diff
@@ -131,7 +135,27 @@ static void SetCaptureType(const std::string_view capture_str)
 	}
 }
 
-static void SetPs2Model(const std::string_view model_str)
+static void set_dos_driver_model(const std::string_view model_str)
+{
+	auto new_model = mouse_config.model_dos;
+
+	if (model_str == model_dos_2button_str) {
+		new_model = MouseModelDos::TwoButton;
+	} else if (model_str == model_dos_3button_str) {
+		new_model = MouseModelDos::ThreeButton;
+	} else if (model_str == model_dos_wheel_str) {
+		new_model = MouseModelDos::Wheel;
+	} else {
+		assert(false);
```

Comment:
Suggest to use `assertm`, e.g.:

```ini
assertm(false, "Invalid mouse model value");
```

Reply:
Changed, also in other places in the file.